cms/envs/common.py

```diff
@@ -1587,6 +1611,19 @@
 
 EVENT_TRACKING_SEGMENTIO_EMIT_WHITELIST = []
 
+############# Cross-domain requests #################
+
+if FEATURES.get('ENABLE_CORS_HEADERS'):
+    CORS_ALLOW_CREDENTIALS = True
+    CORS_ORIGIN_WHITELIST = ()
+    CORS_ORIGIN_ALLOW_ALL = False
+    CORS_ALLOW_HEADERS = corsheaders_default_headers + (
+        'use-jwt-cookie',
+    )
+
+CORS_ORIGIN_WHITELIST = []
+CORS_ORIGIN_ALLOW_ALL = False
```

Comment:
These lines look like they're overwriting the configs defined above

Reply:
https://github.com/edx/edx-platform/blob/a5b42cfc96ef2476f0fffead812acb3e549cd1f2/cms/envs/common.py#L2080-L2081

Reply:
Yeah, this is weird, but it's also copied directly from `lms/envs/common.py`. I bet it's just an artifact of poor settings file hygiene on our part. I'll circle back on this point.

Reply:
So, given that we purposely decoupled the LMS and Studio settings files, I recommend cleaning up cruft like this in anything that you copy over instead of trying to exactly mirror the LMS settings.

Reply:
I don't think anyone will complain if any clean-up done here is also copied back to the LMS settings. ;)

Reply:
CORS support was merged via https://github.com/edx/edx-platform/pull/24571.  I just added a commit with a couple of clean-ups.